src/irmin-pack/inode.ml

```diff
@@ -47,7 +47,9 @@ module type S = sig
 
   val close : 'a t -> unit Lwt.t
 
-  val sync : 'a t -> unit
+  val sync : 'a t -> clear_lrus:(unit -> unit) -> unit
```

Comment:
```suggestion
  val sync : ?on_generation_change:(unit -> unit) -> 'a t -> unit
```

I think `clear_lrus` isn't very descriptive about when this callback is actually triggered. Would be nice to have some documentation in the `.mli` too.

Reply:
Added this [comment](https://github.com/mirage/irmin/pull/1015/files#diff-96f783eaf7be0dd96c1d69871124967aR61-R66), not sure it is very clear though.

Reply:
Clear enough for now; thanks :)